### PR TITLE
Updated bmapi.py and bmutils.py to allow you to specify player (the first player) and description

### PR DESCRIPTION
**Pull request by irilyth**
http://jenkins.buttonweavers.com:8080/job/buttonmen-irilyth/39/ (if it passes)

### CODE REVIEW

**Comment by cgolubi1** [comment]
Hmm, it's a little gross that the defaults for `bmapi.create_game()` are all None and the defaults for `bmutils.wrap_create_game()` are all `''`.  You didn't do that, but any interest in standardizing it while you're here?  Is it obvious which way it should be standardized?

**Comment by irilyth** [comment]
Done! http://jenkins.buttonweavers.com:8080/job/buttonmen-irilyth/40/ (if it passes).

File: tools/api-client/python/lib/bmutils.py [hunk 1]
```diff
@@ -94,11 +94,11 @@ def wrap_load_completed_games(self):
...
 
-  def wrap_create_game(self, pbutton, obutton='', opponent=''):
-    retval = self.create_game(pbutton, obutton, opponent)
+  def wrap_create_game(self, pbutton, obutton='', player='', opponent='', description=''):
+    retval = self.create_game(pbutton, obutton, player, opponent, description)
     if not retval.status == 'ok':
-      raise ValueError, "Failed to call createGame, got: " + retval
-    return retval.data
+      raise ValueError, "Failed to call createGame, got: " + retval.message
+    return retval
```
Comment (cgolubi1) [inline tools/api-client/python/lib/bmutils.py]: Huh, sorry, i didn't notice this change yesterday (from returning `retval.data` to returning `retval`).  That's going to change the expectations of callers if any, isn't it?  And i don't think there's anything interesting in `retval.message` on success.

What does this change get you that you want?

**Comment by irilyth** [comment]
As discussed on Skype, let's leave the return value alone. Fixed in 441cc6b, http://jenkins.buttonweavers.com:8080/job/buttonmen-irilyth/41/ if it passes.

**Comment by cgolubi1** [comment]
Merging.

**Comment by irilyth** [comment]
Done! http://jenkins.buttonweavers.com:8080/job/buttonmen-irilyth/42/ (if it passes)

**Pull request merged**